docs/guide/convert.md

```
@@ -678,12 +678,18 @@ Validates lat/lon ranges (-90 to 90, -180 to 180). Warns on large coordinates su
 
 ### Invalid Geometries
 
-Fails on invalid WKT by default. Skip with `--skip-invalid`:
+Fails on invalid WKT by default. Skip the unparsable rows with
+`--skip-invalid`:
 
 ```bash
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)